core/main_test.go

```diff
@@ -165,11 +166,63 @@ func ExampleRun_config() {
 	//    setgasprice  Set the default gas price to use for outgoing transactions
 	//    loglevel     Set log level
 	//    logsql       Enable/disable sql statement logging
+	//    validate     Validate provided TOML config file
 	//
 	// OPTIONS:
 	//    --help, -h  show help
 }
 
+func ExampleRun_config_validate_without_file() {
+	runEx("config", "validate")
+	// Output:
+	// NAME:
+	//    core.test config validate - Validate provided TOML config file
+	//
+	// USAGE:
+	//    core.test config validate [command options] [arguments...]
+	//
+	// OPTIONS:
+	//    --file FILE, -f FILE  TOML config FILE name to validate
+}
+
+func ExampleRun_config_validate_invalid_file() {
+	runEx("config", "validate", "-f", "/does/not/exist")
+	// Output:
+	// Validation failed (open /does/not/exist: no such file or directory)
+}
+
+func ExampleRun_config_validate_invalid_toml_syntax() {
+	tmpFile, _ := os.CreateTemp("", "")
+	defer tmpFile.Close()
+	tmpFile.WriteString("{invalid syntax {")
+
+	runEx("config", "validate", "-f", tmpFile.Name())
+	// Output:
+	// Validation failed ((1, 1): parsing error: keys cannot contain { character)
+}
+
+func ExampleRun_config_validate_invalid_duplicate_field() {
+	tmpFile, _ := os.CreateTemp("", "")
+	defer tmpFile.Close()
+	tmpFile.WriteString("Dev = false\n")
+	tmpFile.WriteString("Dev = true\n")
```

Comment:
Would it be cleaner to use back quotes?
```suggestion
	tmpFile.WriteString(`Dev = false
Dev = true
`)
```
Also, I wondering about Examples vs. Tests, since these are a little different than the surrounding Examples. Is there a compelling reason to have them be Examples? Seems like a candidate for classic table driven tests :shrug: Also `https://pkg.go.dev/testing#T.TempDir` is convenient.